### PR TITLE
FIX tailwindcss standalone build on monorepos

### PR DESCRIPTION
**Pull request by AlbertSabate**
This fixes the standalone build for tailwindcss when used on a monorepo, prevents having broken symlinks from the folder ".bin".

### CODE REVIEW

File: packages/brisa-tailwindcss/index.ts [hunk 1]
```diff
@@ -49,7 +49,13 @@ export default function brisaTailwindcss({ embedded = true } = defaultConfig) {
           const to = path.join(destNodeModules, lib);
 
           if (await fs.exists(from)) {
-            return fs.cp(from, to, { recursive: true });
+            const toBinPath = path.join(to, 'node_modules/.bin');
```
Comment (aralroca) [inline packages/brisa-tailwindcss/index.ts]:
```suggestion
            const toBinPath = path.join(to, 'node_modules', '.bin');
```

This is to work on Windows that have a different separator than `/`.

Comment (AlbertSabate) [reply packages/brisa-tailwindcss/index.ts]: Uhh, windows. You are right!

**Review by aralroca — CHANGES_REQUESTED** [review]
I wonder if instead of:

- Copy
- Detect and if it exists .bin delete it

Could we directly just copy if `from` does not contain `node_modules/.bin`?

It would also be nice to add the tests for this change.

**Comment by AlbertSabate** [comment]
How the logic is now no. Reason is because we are copying the whole path recursively, so we don't have information about the inner folder. And we need to copy the dependency.
A solution it will be to copy file by file and then ignore .bin folders, but it will be a more complicated logic to read and not sure the benefit is substantiable enough

**Comment by AlbertSabate** [comment]
I will have no issues doing how you say, but I don't see too much benefit.

I'll add a test, do not merge yet.

**Comment by aralroca** [comment]
> I will have no issues doing how you say, but I don't see too much benefit.
> 
> I'll add a test, do not merge yet.

Less work... Copy + detect + delete are 3 operations, detect and avoid copy would be 1.

**Review by aralroca — APPROVED** [review]

**Pull request merged**